packages/amplify_core/lib/src/types/analytics/hub/analytics_hub_event.dart

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_core/amplify_core.dart';
+
+/// {@template amplify_common.hub.analytics_hub_event_type}
+/// Hub Event types for the Analytics category.
+/// {@endtemplate}
+enum AnalyticsHubEventType {
+  /// {@macro amplify_common.hub.analytics_hub_event_pinpoint_endpoint_id_set}
+  pinpointEndpointIdSet('PINPOINT_ENDPOINT_ID_SET');
```

Comment:
Not in love with this name - it implies a non-fixed ID and the existence of an `unset` event as well.. what about just `configured` or something since it will be fired every startup after configuration completes?

Reply:
Sounds good!  Made the change to -> PINPOINT_ENDPOINT_ID_CONFIGURED

I think ID is a bit wordy but it conveys that we're talking specifically about ID being configured here and nothing else as the scope is quite focused on the endpoint id being configured.